### PR TITLE
client statistics code sample

### PR DESCRIPTION
**Pull request by asimarslan**

### CODE REVIEW

File: Hazelcast.Examples/Client/ClientStatisticsExample.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+{
+    internal class ClientStatisticsExample
+    {
+        private static void Run(string[] args)
+        {
+            //This will enable client statistics 
+            Environment.SetEnvironmentVariable("hazelcast.client.statistics.enabled", "true");
+            
+            //set the statistics send period, default value is 3 seconds
+            Environment.SetEnvironmentVariable("hazelcast.client.statistics.period.seconds", "3");
```
Comment (ihsandemir) [inline Hazelcast.Examples/Client/ClientStatisticsExample.cs]: you may use a non-default value for the example.

File: Hazelcast.Examples/Client/ClientStatisticsExample.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            
+            var config = new ClientConfig();
+            config.GetNetworkConfig().AddAddress("127.0.0.1");
```
Comment (ihsandemir) [inline Hazelcast.Examples/Client/ClientStatisticsExample.cs]: do we need this line? don't we add 127.0.0.1 by default if the list is empty?

File: Hazelcast.Examples/Client/ClientStatisticsExample.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            var client = HazelcastClient.NewHazelcastClient(config);
+            //After client connected you can use Management Center to visualize client statistics
+
```
Comment (ihsandemir) [inline Hazelcast.Examples/Client/ClientStatisticsExample.cs]: you may put some sleep here to give the user some time to visualise the statistics.

File: Hazelcast.Examples/Client/ClientStatisticsExample.cs (hunk 2 above)
Comment (asimarslan) [reply Hazelcast.Examples/Client/ClientStatisticsExample.cs]: no we need it, there isn't any default

File: Hazelcast.Examples/Client/ClientStatisticsExample.cs (hunk 3 above)
Comment (asimarslan) [reply Hazelcast.Examples/Client/ClientStatisticsExample.cs]: Let me put some client activity instead

File: Hazelcast.Examples/Client/ClientStatisticsExample.cs (hunk 2 above)
Comment (ihsandemir) [reply Hazelcast.Examples/Client/ClientStatisticsExample.cs]: this behaviour seems to differ from Java client.

**Pull request merged**